### ISSUE TITLE
[1.19.2] Incompatible with Vinery

### ISSUE THREAD

**Issue by SleepyEmber** [LambdAurora/AurorasDecorations#43]
I need to try this isolated as well, but after some testing, Aurora's Decorations seems to be incompatible with Vinery
https://paste.ee/p/v91AN
https://www.curseforge.com/minecraft/mc-mods/lets-do-wine

**Comment by Sunconure11** [comment]
Have this issue too. @LambdAurora

**Comment by Sunconure11** [comment]
Found the class that is probably conflicting.

https://github.com/satisfyu/Vinery/blob/e9c98dbdf2051158b45deebfcf088b753d7c5637/common/src/main/java/satisfyu/vinery/block/StackableLogBlock.java#L38

**Comment by Sunconure11** [comment]
https://pastebin.com/3kyAjsN1

Forgot to post a fresh crash report

**Comment by SplendidAlakey** [comment]
The mods are compatible on 1.20.x now, but I'm not sure which side is responsible and if the changes could be backported.

**Comment by Sunconure11** [comment]
Yeah, I saw that they are on 1.20.x and 1.19.4 now. However, I can't close this issue so I will leave it up to @LambdAurora to decide if they want to backport the fix or not, and if they want to close this issue.

**Comment by LambdAurora** [comment]
ngl, I don't like backporting stuff.

**Comment by Sunconure11** [comment]
Alright, can you close this issue then?
